lib/WGmeta/Wireguard/Wrapper/Bridge.pm

```diff
@@ -0,0 +1,104 @@
+package WGmeta::Wireguard::Wrapper::Bridge;
+use strict;
+use warnings FATAL => 'all';
+use experimental 'signatures';
+
+use base 'Exporter';
+our @EXPORT = qw(gen_keypair get_wg_show);
+
+use Symbol 'gensym';
+use IPC::Open3;
+
+use constant FALSE => 0;
+use constant TRUE => 1;
+
+
+=head3 gen_keypair()
+
+Runs the C<wg genkey> command and returns a private and the corresponding public-key
+
+B<Returns>
+
+Two strings private-key and public-key
+
+=cut
+sub gen_keypair() {
+    my $cmd = 'priv_key=$(wg genkey) && echo $priv_key | wg pubkey && echo $priv_key';
```

Comment:
iiks

Reply:
since you are using open3 it seems odd to then still use tonns of shell magic ... I would rather call wg genkey and wg pubkey in two separate calls

Reply:
This is indeed a bit hacky. "Upgraded" `run_external()` to accept a string feed to STD_IN of the cmd